realsense2_camera/src/base_realsense_node.cpp

```diff
@@ -3,6 +3,24 @@
 
 using namespace realsense2_camera;
 
+/* Modified for Brain */
+#include <stdio.h>
+#define FRAME_MAX_HEIGHT 720
```

Comment:
For C++, in general you want to prefer declaring constant values with _const static_ over using preprocessor _defines_